### ISSUE TITLE
v1.78a (Windows): Nexapow broken

### ISSUE THREAD

**Issue by UselessGuru** [Lolliedieb/lolMiner-releases#1984]
`Bin\lolMiner-v1.78a\lolminer.exe --algo NEXA --pool acc-pool.pw:16011 --user nexa:nqtsq5g5galktrng3lfa3u9p7yv9d0dzzkv400vgaz23m77e --tls off --log off --apiport 4004 --shortstats 1 --longstats 5 --digits 6 --watchdog exit --dns-over-https 1 --devicesbypcie --devices 9:0`
```
+---------------------------------------------------------+
|   _       _ __  __ _                   _   _____  ___   |
|  | | ___ | |  \/  (_)_ __   ___ _ __  / | |___  |( _ )  |
|  | |/ _ \| | |\/| | | '_ \ / _ \ '__| | |    / / / _ \  |
|  | | (_) | | |  | | | | | |  __/ |    | |_  / / | (_) | |
|  |_|\___/|_|_|  |_|_|_| |_|\___|_|    |_(_)/_/   \___/  |
|                                          Bugfix Version |
|                                                         |
|              This software is for mining                |
|              Autolykos V2                               |
|              Ethash, Etchash                            |
|              Equihash 144/5, 192/7, 210/9               |
|              BeamHash III                               |
|              NexaPow                                    |
|              Flux (ZelHash)                             |
|              Heavyhash-Karlsen                          |
|              Blake3 (Ironfish & Alephium)               |
|              Cuck(ar)oo 29                              |
|              Cuckaroo   30 CTX                          |
|              Cuckatoo   31/32                           |
|                                                         |
|                                                         |
|            Made by Lolliedieb, December 2023            |
+---------------------------------------------------------+

Setup Miner...
OpenCL driver detected.
Number of OpenCL supported GPUs: 2
Cuda driver detected.
Number of Cuda supported GPUs: 3
Device 0:
    Name:    Radeon RX 580 Series
    Address: 1:0
    Vendor:  Advanced Micro Devices (AMD)
    Drivers: OpenCL
    Memory:  8192 MByte (8128 MByte free)
    Active:  false (not selected)
Device 1:
    Name:    AMD Radeon RX 5700 XT
    Address: 9:0
    Vendor:  Advanced Micro Devices (AMD)
    Drivers: OpenCL
    Memory:  8176 MByte (8112 MByte free)
    Active:  true (Selected Algorithm: NexaPow (sha256d + Secp256k1))
Device 2:
    Name:    NVIDIA GeForce GTX 1660 SUPER
    Address: 11:0
    Vendor:  NVIDIA Corporation
    Drivers: Cuda
    Memory:  6143 MByte
    Active:  false (not selected)
Device 3:
    Name:    NVIDIA GeForce GT 1030
    Address: 12:0
    Vendor:  NVIDIA Corporation
    Drivers: Cuda
    Memory:  2047 MByte
    Active:  false (not selected)
Device 4:
    Name:    NVIDIA GeForce GTX 750 Ti
    Address: 16:0
    Vendor:  NVIDIA Corporation
    Drivers: Cuda
    Memory:  2047 MByte
    Active:  false (not selected)

Applying overclock settings...

Connecting to pool...
DNS over HTTPS resolve succeeded
Connected to acc-pool.pw(162.55.7.43):16011  (TLS disabled)
Subscribed to stratum server
New extra nonce received: f6be1cec
Authorized worker: nexa:nqtsq5g5galktrng3lfa3u9p7yv9d0dzzkv400vgaz23m77e
New target received: 00000009fffffff6 (Diff 0.1)
New work received: 3b
Start Mining...
Average speed (1s): 0.000000 Mh/s
Average speed (1s): 0.000000 Mh/s
```

Boom. Miner exits without error message.
I tested with GTX 1660 Super, RX 580 & RX5700.
I also tried miningnexa @ ZergPool & Nicehash. All with the same result.

**Comment by angelbbs** [comment]
Error message:
`Start Mining...
GPU 0: error in allocating memory 0 .
0x00007FF7C5636801 (0x00005EB67D2A8194 0x00005EB67D2A8194 0x0000000000000003 0x000001979CDFC6A0)
0x00007FF7C565EDFD (0x0000000000000057 0x0000019794B39820 0x0000000000000000 0x0000000000000000)
0x00007FF7C5613023 (0x00000197A8C76680 0x0000000000000000 0x0000000000000000 0x0000000000000000)
0x00007FF7C59ACE08 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000)
0x00007FFA4B1A7974 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000), BaseThreadInitThunk() + 0x14 bytes(s)
0x00007FFA4B55A271 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000), RtlUserThreadStart() + 0x21 bytes(s)`

**Comment by UselessGuru** [comment]
Fixed with 1.79
